### PR TITLE
7 post api articles article id comments

### PR DESCRIPTION
**Pull request by ezkl123**

### CODE REVIEW

File: __tests__/api.test.js [hunk 1]
```diff
@@ -153,19 +153,92 @@ describe('/api/articles/:article_id/comments testing', () => {
...
+describe('POST /api/articles/:article_id/comments', () => {
+    test('returns 200 with the posted comment', () => {
+        return request(app)
+        .post('/api/articles/2/comments')
+        .send({
+            username: 'butter_bridge',
+            body: 'What a cool article'
+        })
+        .expect(200)
+        .then((response) => {
```
Comment (nc-jodie-cheung) [inline __tests__/api.test.js:173]: You would be expecting a status 201 for a POST 👀

File: __tests__/api.test.js [hunk 2]
```diff
@@ -153,19 +153,92 @@ describe('/api/articles/:article_id/comments testing', () => {
...
+            const comment = response.body.comment[0];
+            expect(comment.body).toBe('What a cool article')
+        })
+    })
```
Comment (nc-jodie-cheung) [inline __tests__/api.test.js:177]: I'd consider adding in a test for another Status 201 - that ignores unnecessary properties. For example:

```
({
            username: 'butter_bridge',
            body: 'What a cool article',
            extraProperty: 'extra'
        })
```

File: controllers/controllers.js [hunk 3]
```diff
@@ -55,4 +55,18 @@ function sendAllComments(req, res, next){
...
+    const {article_id} = req.params;
+    const {username, body} = req.body;
+    // console.log(username)
+    return addComments(username, body, article_id)
+    .then((comment) => {
+        res.status(200).send({comment})
+    })
+    .catch((err) => {
+        next(err)
+    })
```
Comment (nc-jodie-cheung) [inline controllers/controllers.js:69]: Don't forget to remove console logs from your code. Keep it looking nice and tidy!

**Review by nc-jodie-cheung — COMMENTED** [review]
Nice work, please check out my comments 👍

File: __tests__/api.test.js (hunk 1 above)
Comment (ezkl123) [reply __tests__/api.test.js:173]: yep, forgot about that, I fixed it just now. thanks!

**Pull request merged**